onnxruntime/python/tools/kernel_explorer/kernels/gemm_fast_gelu_test.py

```diff
@@ -0,0 +1,197 @@
+    beta = 0.0
+    my_func = getattr(ke, func)
+    my_op = my_func(opa, opb, m, n, k, alpha, dev_a, lda, dev_b, ldb, dev_bias, beta, dev_c, n)
+
+    if my_op.IsSupported():
+        my_op.Run()
+        dev_c.UpdateHostNumpyArray()
+
+        print(
+            f"{func:<50} : dtype={dtype} {transab_to_suffix((transa, transb))} m={m:<5} n={n:<5} k={k:<5} bound: {bound}"
+        )
+        try:
+            np.testing.assert_allclose(my_c, ref_c, rtol=max(bound, 1e-2))
+        except Exception as err:
+            header = "*" * 30 + func + "*" * 30
+            print(header)
+            print(err)
+            print("*" * len(header))
+            raise Exception(str(err))
+
+
+dtypes = ["float16", "float32"]
+all_transabs = list(product([True, False], repeat=2))
+all_basic_sizes = list(product([1, 3, 4, 16, 127, 128, 129, 133, 1024], repeat=3))
+
+
+def get_bert_sizes(full=True):
+    bert_base_sizes = [
+        # m, n, k
+        (384, 768, 768),
+        (384, 768, 768 * 3),
+        (384, 768, 768 * 4),
+        (384, 768 * 4, 768),
+        (384, 1024, 1024),
+        (384, 1024, 1024 * 3),
+        (384, 1024, 1024 * 4),
+        (384, 1024 * 4, 1024),
+    ]
+
+    # we then multiply m with the batch size
+    if full:
+        batch_sizes = [1, 64]
+    else:
+        batch_sizes = [1]
+    bert_sizes = []
+    for bsz in batch_sizes:
+        bert_sizes.extend([(m * bsz, n, k) for m, n, k in bert_base_sizes])
+    return bert_sizes
+
+
+reduced_basic_sizes = list(product([1, 4, 127, 133], [3, 16, 128], [3, 129, 1024]))
+
+
+@pytest.mark.parametrize("dtype", dtypes)
+@pytest.mark.parametrize("size", reduced_basic_sizes + get_bert_sizes(full=False))
+@pytest.mark.parametrize("transab", all_transabs)
+def test_gemmfastgelu_bert_cases(dtype, size, transab):
+    for func in dtype_to_funcs(dtype):
+        _test_gemmfastgelu(func, dtype, *size, *transab)
+
+
+def profile_gemmfastgelu_func(func, dtype: str, m: int, n: int, k: int, transa: bool, transb: bool):
+    a_shape = (k, m) if transa else (m, k)
+    b_shape = (n, k) if transb else (k, n)
+
+    np.random.seed(0)
+    a = (np.random.rand(*a_shape) * 2 - 1).astype(dtype)
+    b = (np.random.rand(*b_shape) * 2 - 1).astype(dtype)
+    my_c = np.zeros((m, n), dtype=dtype)
+    bias = np.random.rand(n).astype(dtype)
+
+    dev_a = ke.DeviceArray(a)
+    dev_b = ke.DeviceArray(b)
+    dev_bias = ke.DeviceArray(bias)
+    dev_c = ke.DeviceArray(my_c)
+
+    opa = ke.blas_op.T if transa else ke.blas_op.N
+    opb = ke.blas_op.T if transb else ke.blas_op.N
+    lda = a_shape[1]
+    ldb = b_shape[1]
+    alpha = 1.0
+    beta = 0.0
+    my_func = getattr(ke, func)
+    my_op = my_func(opa, opb, m, n, k, alpha, dev_a, lda, dev_b, ldb, dev_bias, beta, dev_c, n)
+
+    if my_op.IsSupported():
+        my_op.Run()
+        dev_c.UpdateHostNumpyArray()
+
+        time_ms = my_op.Profile()
+        time_us = time_ms * 1000
+        tflops = (m * k * n * 2) / (time_ms * 1e-3) / 1e12
```

Comment:
better leave some comment about this tflops only counts gemm part due to gelu part is low order term.